stacks/kafka-flink/README.Unix.rst

```diff
@@ -13,6 +14,22 @@ This document will outline how to run the tutorial on Linux, macOS and WSL2.
 Setup
 *****
 
+Prerequisites
+=============
+
+In order to exercise the command walkthrough successfully, please make sure
+those programs are installed/available on your machine / in your environment.
+
+- bash
+- cat
+- curl
+- docker
+- git
+- head
+- jq
+- tar
```

Comment:
Maybe it's important to mention that is gnu tar, not bsd tar (default tar in MacOS)?

Reply:
Maybe BSD tar works in this case equally well?

Reply:
if a tarball is created with gnu tar then the bsd tar most probably won't be able to extract it.

Reply:
It works.

```
tar -xzf nyc-yellow-taxi-2017.tar.gz
```
```
tar --version
bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6
```

Reply:
ok, cool, thx for checking, I have encountered problems with gnu vs bsd in the past, https://xorl.wordpress.com/2012/05/15/admin-mistakes-gnu-bsd-tar-and-posix-compatibility/